Tools/MiniBrowser/gtk/BrowserWindow.c

```diff
@@ -482,6 +482,20 @@ static GtkWidget *webViewCreate(WebKitWebView *webView, WebKitNavigationAction *
     return GTK_WIDGET(newWebView);
 }
 
+void browser_window_fullscreen(BrowserWindow *window)
+{
+    gtk_window_fullscreen(GTK_WINDOW(window));
+    gtk_widget_hide(window->toolbar);
+    window->fullScreenIsEnabled = TRUE;
+}
+
+void browserWindowUnfullscreen(BrowserWindow *window)
```

Comment:
OK, one last change: this file-scope function should not be exported:

```suggestion
static void browserWindowUnfullscreen(BrowserWindow *window)
```